templates/user/settings/keys_ssh.tmpl

```diff
@@ -1,8 +1,10 @@
 <h4 class="ui top attached header">
   {{.i18n.Tr "settings.manage_ssh_keys"}}
+  {{if not .DisableSSH}}
```

Comment:
I would also hide this entire tab TBH. Or at least should a warning here that SSH is disabled 🙂